analytics_data_api/v0/views/problems.py

```diff
@@ -55,6 +56,48 @@ class ProblemResponseAnswerDistributionView(generics.ListAPIView):
     serializer_class = ConsolidatedAnswerDistributionSerializer
     allow_empty = False
 
+    @classmethod
```

Comment:
I love improvements like this! Thanks for taking a moment to leave the code better than you found it.